.github/workflows/hqs-build-deploy-sphinx-doc-rust-pyo3.yml

```diff
@@ -31,7 +31,11 @@ jobs:
           python -m sphinx -T -E -b html . _build/html
           mv _build/html/* ../../documentation/src/python_api_docs
           cd ../..
-          mdbook build documentation
+          cargo install mdbook-cmdrun
+          cargo install cargo-single
+          cd documentation
+          mdbook build
+          cd ..
```

Comment:
Is this last line needed?

Reply:
I believe so, otherwise the publishing job may end up having path problems.